corehq/apps/data_cleaning/tests/test_models.py

```diff
@@ -0,0 +1,103 @@
+from django.test import TestCase
+
+from casexml.apps.case.mock import CaseFactory
+from corehq.apps.data_cleaning.models import (
+    BulkEditChange,
+    EditActionType,
+)
+from corehq.form_processor.tests.utils import FormProcessorTestUtils
+
+
+class BulkEditChangeTest(TestCase):
+    domain = 'planet'
+    case_type = 'island'
+
+    def setUp(self):
+        factory = CaseFactory(domain=self.domain)
+        self.case = factory.create_case(
+            case_type=self.case_type,
+            owner_id='q123',
+            case_name='Vieques',
+            update={
+                'nearest_ocean': 'atlantic',
+                'town': 'Isabel Segunda',
+                'favorite_beach': 'Playa Bastimento',
+                'art': '  :)     ',
+                'second_favorite_beach': 'no idea',
+            },
+        )
+
+        super().setUp()
+
+    def tearDown(self):
+        FormProcessorTestUtils.delete_all_cases()
+        super().tearDown()
+
+    def test_replace(self):
+        change = BulkEditChange(
+            property='town',
+            action_type=EditActionType.REPLACE,
+            replace_string='Esperanza',
+        )
+        self.assertEqual(change.edited_value(self.case), 'Esperanza')
+
+    def test_find_replace(self):
+        change = BulkEditChange(
+            property='favorite_beach',
+            action_type=EditActionType.FIND_REPLACE,
+            find_string='Playa',
+            replace_string='Punta',
+        )
+        self.assertEqual(change.edited_value(self.case), 'Punta Bastimento')
```

Comment:
should also add a test for regular expressions

Reply:
(a different test where `use_regex=True`)

Reply:
Not implementing regular expressions in this PR.